coordinator/src/escrow_coordinator/mod.rs

```diff
@@ -56,16 +56,16 @@ impl EscrowCoordinator {
                                     EscrowCoordinator::parse_contract(&rumor.content)
                                 {
                                     debug!("Received contract: {}", &contract.trade_description);
-                                    if self.pending_contracts.contains_key(&contract_hash) {
+                                    if let std::collections::hash_map::Entry::Vacant(e) = self.pending_contracts.entry(contract_hash) {
```

Comment:
What about making an import for Entry and writing just `Entry::Vacant(e)`?

Reply:
Agree, looks way cleaner